var/spack/repos/builtin/packages/nmap/package.py

```diff
@@ -46,11 +45,16 @@ def configure_args(self):
         # https://github.com/nmap/nmap/issues/2144
         args.append("--disable-rdma")
 
+        # ndiff and zenmap both require python2, which is deprecated.
```

Comment:
Is it worth noting that it is Spack that deprecated `python2` support or is that clear (to others)?

Reply:
yep! added that